.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: kind/bug
+assignees: ''
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
+
+If the matter is security related, please disclose it privately via 
+-->
+
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+- Agones version: 
+- Kubernetes version (use `kubectl version`):
+- Cloud provider or hardware configuration:
+- Install method (yaml/helm):
+- Troubleshooting guide log:
```

Comment:
```suggestion
- Troubleshooting guide log(s):
```

Super minor nit, since it could be controller or sidecar logs.

Reply:
Do we want to link to https://agones.dev/site/docs/guides/troubleshooting/ here?